internal/app/cache/cache.go

```diff
@@ -236,10 +239,29 @@ func (c *cache) DeleteRequest(key string, req transport.Request) error {
 		"node_id", req.GetNodeID(),
 		"request_type", req.GetTypeURL(),
 	).Debug(context.Background(), "request deleted")
-	metrics.CacheAddRequestSubscope(c.scope, key).Counter(metrics.CacheDeleteSuccess).Inc(1)
+	metrics.CacheDeleteRequestSubscope(c.scope, key).Counter(metrics.CacheDeleteSuccess).Inc(1)
 	return nil
 }
 
+func (c *cache) DeleteKey(key string) (Resource, error) {
+	c.cacheMu.Lock()
```

Comment:
we've added tests in `internal/app/admin/http/cache_test.go` . Should we add tests for this function as well for completeness?